README.md

```diff
@@ -22,11 +22,54 @@ The goal of this package is to offer almost all the features available on the we
 
 3. Login with your credentials and move to the profile section.
 
-4. Click on `Get your Auth Token` to get the authentication token & download it as a JSON file.
+4. To setup the auth token, run the following command:
+```
+evalai set_token <token>
+```
+
+### Step 2:
+
+1. Clone the evalai-cli repository to your machine via git
+
+```bash
+git clone https://github.com/Cloud-CV/evalai-cli.git EvalAI-CLI
+```
+
+2. Create a virtual environment
 
-5. Create a folder with name `.evalai` in your home directory by using the command `mkdir ~/.evalai`.
+```bash
+cd EvalAI-CLI
+virtualenv -p python3 venv
+source venv/bin/activate
+```
 
-6. Place the downloaded authentication token in this folder by using the command `mv ~/Downloads/token.json ~/.evalai/`.
+3. Install the package dependencies
+
+```bash
+pip install -r requirements.txt
+```
+
+4. Install the package locally to try it out
+
+```bash
+pip install -e .
+```
+
+## Development Setup for Windows
+
+### Step 1:
+
+1. Setup the development environment for EvalAI and make sure that it is running perfectly.
+
+2. Now, Open web browser and hit the url http://127.0.0.1:8888.
+
+3. Login with your credentials and move to the profile section.
+
+4. To setup the auth token, run the following command:
```

Comment:
This command will work only once `evalai` package is setup. Please move it after the step 4.